tests/mfx_dispatch/linux/mfx_dispatch_test_cases_plugins.cpp

```diff
@@ -59,19 +59,20 @@ TEST_P(DispatcherPluginsTestParametrized, ShouldLoadFromGoodPluginsCfgCorrectly)
         // and the parsed plugin list is stored as a global variable in the
         // dispatcher lib. Therefore, the calls below will only be executed once.
         std::string path = std::string(MFX_PLUGINS_CONF_DIR) + "/plugins.cfg";
-        EXPECT_CALL(mock, fopen(StrEq(path.c_str()),_)).WillRepeatedly(Return(MOCK_FILE_DESCRIPTOR));
-        EXPECT_CALL(mock, fgets(_,_,MOCK_FILE_DESCRIPTOR)).WillRepeatedly(Invoke(&mock, &MockCallObj::FeedEmulatedPluginsCfgLines));
-        EXPECT_CALL(mock, fclose);
+        EXPECT_CALL(mock, fopen(StrEq(path.c_str()), _)).Times(AtLeast(1)).WillRepeatedly(Return(MOCK_FILE_DESCRIPTOR));
```

Comment:
I think we open plugins.cfg only once. Shouldn't we expect fopen/fclose `Times(1)`?

Reply:
You are right. It will be replaced.